### DISCUSSION TITLE
Welcome to pi-transceiver Discussions!

### DISCUSSION THREAD

**Discussion by M0LTE** [M0LTE/pi-transceiver#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by G4CDF** [comment]
Hi, I am Mike G4CDF and will help with the RF design of this project.  This starts with a bandpass filter, transmitter power amplifier, receiver low noise amplifier and antenna changeover for the 144MHz band.
